build/windows/fix_archives.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+#
+#
+# Gets dist/zip_dirty created by Goreleaser and reorganize inside files
+#
+#
+PROJECT_PATH=$1
+
+for zip_dirty in $(find dist -regex ".*_dirty\.\(zip\)");do
+  zip_file_name=${zip_dirty:5:${#zip_dirty}-(5+10)} # Strips begining and end chars
```

Comment:
`5` and `5+10` offsets might be problematic in the near future.

Reply:
feel free to create variables or another solution 🙂